File: php/3rd_week/latihanpraktikum3_php/MainDevice.php
<?php

include "Pc.php";

// instansiasi class processor
$dataProc = new Processor();
$dataProc->setName("Intel");
$dataProc->setProcPrice(3200000);

// instansiasi class disk
$dataDisk = new Disk();
$dataDisk->setType("SSD");
$dataDisk->setDiskCapacity("512 GB");
$dataDisk->setDiskPrice(800000);

// instansiasi class ram
$dataRam = new Ram();
$dataRam->setRamCapacity("16 GB");
$dataRam->setRamPrice(1200000);

// instansiasi class pc
$dataPc = new Pc();
$dataPc->setProcObj($dataProc);
$dataPc->setDiskObj($dataDisk);
$dataPc->setRamObj($dataRam);

$dataPc->countTotal();      //method hitung total price
$dataPc->printPc();         //method menampilkan semua data pc

?>

File: python/3rd_week/latihanpraktikum3_py/MainDevice.py
from Processor import Processor
from Disk import Disk
from Ram import Ram
from Pc import Pc

# instansiasi class processor

dataProc = Processor()
dataProc.setName("AMD")
dataProc.setProcPrice(2500000)

# instansiasi class disk
dataDisk = Disk()
dataDisk.setDiskType("HDD")
dataDisk.setDiskCapacity("1 TB")
dataDisk.setDiskPrice(500000)

# instansiasi class ram
dataRam = Ram()
dataRam.setRamCapacity("GB")
dataRam.setRamPrice(1500000)

# instansiaasi class pc
dataPc = Pc()
dataPc.setProcObj(dataProc)
dataPc.setDiskObj(dataDisk)
dataPc.setRamObj(dataRam)

# memanggil method hitung total price
dataPc.countTotal()

# print seluruh data pada objek
dataPc.printPc()